extensions/amp-viewer-assistance/0.1/test/validator-amp-viewer-assistance.html

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
This file already exists as [validator-actions-amp-viewer-assistance.html](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-assistance/0.1/test/validator-actions-amp-viewer-assistance.html)

Reply:
Removed.